<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>79</x>
      <y>319</y>
      <width>141</width>
      <height>151</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="counter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>241</width>
      <height>37</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>American Typewriter</family>
      <pointsize>45</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>650</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/blackBackground.jpg</pixmap>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>590</y>
      <width>314</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="menu">
     <property name="spacing">
      <number>-1</number>
     </property>
     <property name="leftMargin">
      <number>8</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item>
      <widget class="QPushButton" name="homeButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="marketButton">
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="passiveIncomeButton">
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="giveMoneyButton">
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="hamsterLVL">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="coinsPerClickLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>American Typewriter</family>
     </font>
    </property>
    <property name="text">
     <string>Coins/click: 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="give1000Button">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>120</y>
      <width>191</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="give5000Button">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>240</y>
      <width>191</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="give10000Button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>191</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="coinsPerSecondLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>American Typewriter</family>
     </font>
    </property>
    <property name="text">
     <string>Coins/sec: 0</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>261</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>221</width>
        <height>441</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="buyItemLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="buyItemButton1">
         <property name="text">
          <string>+1 per click - 50</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buyItemButton2">
         <property name="text">
          <string>+5 per click - 250</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buyItemButton3">
         <property name="text">
          <string>+10 per click - 500</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buyItemButton4">
         <property name="text">
          <string>+20 per click - 1000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buyItemButton5">
         <property name="text">
          <string>+40 per click - 2000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buyItemButton6">
         <property name="text">
          <string>+80 per click - 4000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buyItemButton7">
         <property name="text">
          <string>+160 per click - 8000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buyItemButton8">
         <property name="text">
          <string>+320 per click - 16000</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>221</width>
        <height>441</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="buyPassiveItemLayout">
       <item>
        <widget class="QPushButton" name="buyPassiveButton1">
         <property name="text">
          <string>+5 per second - 250</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buyPassiveButton2">
         <property name="text">
          <string>+10 per second - 500</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buyPassiveButton3">
         <property name="text">
          <string>+20 per second - 1000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buyPassiveButton4">
         <property name="text">
          <string>+40 per second - 2000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buyPassiveButton5">
         <property name="text">
          <string>+80 per second - 4000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buyPassiveButton6">
         <property name="text">
          <string>+160 per second - 8000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buyPassiveButton7">
         <property name="text">
          <string>+320 per second - 16000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buyPassiveButton8">
         <property name="text">
          <string>+640 per second - 32000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buyPassiveButton9">
         <property name="text">
          <string>+1280 per second - 64000</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <zorder>background</zorder>
   <zorder>counter</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>hamsterLVL</zorder>
   <zorder>progressBar</zorder>
   <zorder>coinsPerClickLabel</zorder>
   <zorder>give1000Button</zorder>
   <zorder>give5000Button</zorder>
   <zorder>give10000Button</zorder>
   <zorder>coinsPerSecondLabel</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>pushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>42</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHamster">
    <property name="title">
     <string>Hamster</string>
    </property>
   </widget>
   <addaction name="menuHamster"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
